### PR TITLE
Some documentation for Viewlet subclasses.

### PR DESCRIPTION
**Pull request by shans**

### CODE REVIEW

File: runtime/viewlet.js [hunk 1]
```diff
@@ -87,12 +93,23 @@ class View extends Viewlet {
   query() {
     // TODO: things
   }
+  /** @method toList()
```
Comment (dstockwell) [inline runtime/viewlet.js]: Include `async` in the signatures?

File: runtime/viewlet.js [hunk 2]
```diff
@@ -105,22 +122,45 @@ class View extends Viewlet {
...
+ * the types of views that need to be connected to that particle, and
+ * the current recipe identifies which views are connected.
+ */
 class Variable extends Viewlet {
   constructor(variable, canRead, canWrite) {
     super(variable, canRead, canWrite);
   }
+
+  /** @method get()
+  * Returns the Entity contained by the Variable.
```
Comment (dstockwell) [inline runtime/viewlet.js]: or `undefined`

File: runtime/viewlet.js [hunk 3]
```diff
@@ -79,6 +80,11 @@ class Viewlet {
   }
 }
 
+/** @class View
+ * A handle on a set of Entity data. A particle's manifest dictates
```
Comment (dstockwell) [inline runtime/viewlet.js]: Is `set` enough to imply no duplicates/ordering?

File: runtime/viewlet.js (hunk 1 above)
Comment (shans) [reply runtime/viewlet.js]: Done

File: runtime/viewlet.js (hunk 3 above)
Comment (shans) [reply runtime/viewlet.js]: I think so?

File: runtime/viewlet.js (hunk 2 above)
Comment (shans) [reply runtime/viewlet.js]: done

**Pull request merged**